Makefile

```diff
@@ -31,3 +31,21 @@ outdated: ## Show outdated dependencies
 
 install:  ## Install development dependencies
 	@pip install -r requirements-dev.txt
+
+release-draft: ## Show new release changelog
+	towncrier --draft
+
+release-patch: ## Create patch release
+	bumpversion patch
+	towncrier --yes
+	git commit -am 'Update CHANGELOG' && git push && git push --tags
```

Comment:
I think it is better to not do a `git push` in the release commands, because it is not easily undone and a command may be executed accidentally.